ngraph/core/include/ngraph/runtime/reference/concat.hpp

```diff
@@ -16,68 +16,22 @@
 
```

Comment:
Please move your reference implementation to the [library with reference implementations](https://github.com/openvinotoolkit/openvino/tree/master/ngraph/core/reference).

Reply:
done